packages/components/src/components/form/FractionButton/FractionButton.tsx

```diff
@@ -0,0 +1,39 @@
+import { Button, Tooltip } from '@trezor/components';
+
+import { ButtonVariant } from '../../buttons/buttonStyleUtils';
+
+const DEFAULT_VARIANT = 'tertiary';
+
+export type FractionButtonItem = {
+    label: string;
+    translation?: React.ReactNode;
```

Comment:
I don't understand the need for both label and translation. You don't need a key at this level, I think. I'd just keep children prop.